aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -1366,6 +1375,10 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
         });
 ```
 
+> [!NOTE]
+> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureHttpsDefault*> will not have the defaults applied.
```

Comment:
Double nit:

```suggestion
> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureHttpsDefaults> will not have the defaults applied.
```

There's only one ConfigureHttpsDefaults but Default**s** is plural.

Reply:
Leave the `*` (required) but add the `s` in there.

```
<xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureHttpsDefaults*>
```

Reply:
While ur there, use "won't" over "will not" ... *contractions are good things*. :smile:

Reply:
```suggestion
> Endpoints created by calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.Listen*> **before** calling <xref:Microsoft.AspNetCore.Server.Kestrel.Core.KestrelServerOptions.ConfigureHttpsDefaults*> will not have the defaults applied.
```